11/alpine/hotspot/Dockerfile

```
@@ -62,7 +75,8 @@ ENV JENKINS_UC_EXPERIMENTAL=https://updates.jenkins.io/experimental
 ENV JENKINS_INCREMENTALS_REPO_MIRROR=https://repo.jenkins-ci.org/incrementals
 RUN chown -R ${user} "$JENKINS_HOME" "$REF"
 
-ARG PLUGIN_CLI_URL=https://github.com/jenkinsci/plugin-installation-manager-tool/releases/download/2.9.3/jenkins-plugin-manager-2.9.3.jar
+ARG PLUGIN_CLI_VERSION=2.9.3
```

Intentionally remaining on an older version on Linux?
```suggestion
ARG PLUGIN_CLI_VERSION=2.10.0
```